website/content/en/docs/getting-started/getting-started-with-karpenter/scripts/step08-apply-helm-chart.sh

```diff
@@ -1,11 +1,16 @@
 # Logout of helm registry to perform an unauthenticated pull against the public ECR
 helm registry logout public.ecr.aws
 
+# Set AWS_REGION env variable and annotate Karpenter service account
+# with the IAM Role ARN when using Fargate profile
 helm upgrade --install karpenter oci://public.ecr.aws/karpenter/karpenter --version "${KARPENTER_VERSION}" --namespace "${KARPENTER_NAMESPACE}" --create-namespace \
   --set "settings.clusterName=${CLUSTER_NAME}" \
   --set "settings.interruptionQueue=${CLUSTER_NAME}" \
   --set controller.resources.requests.cpu=1 \
   --set controller.resources.requests.memory=1Gi \
   --set controller.resources.limits.cpu=1 \
   --set controller.resources.limits.memory=1Gi \
+  #  --set controller.env[0].name=AWS_REGION \
```

Comment:
@santosh-at-github Why did you need to set the region? There is a warning section about pod identity and the need to set the service account annotation in the warning for the installing karpenter and using IRSA. The team would like to keep this section clean of comment such that customer can copy and apply helm command. We would like customers to only edit the `helm install` if they are doing something different then the general setup guide.

Reply:
You are right. There is no need to set the region. However the warning section you mentioned, points out that Pod Identity isn't supported on Kubernetes versions < 1.24 and IRSA should be used instead. It doesn't mention anything about Fargate pod. This indicates when running Karpenter pods on Fargate, "Pod Identity" would work, but it doesn't actually. I think it should be explicitly called out in the warning section which is more cleaner way of conveying the same instead what I proposed.

Reply:
Yes, I have made the necessary changes in my latest commit.